llvm/test/Assembler/scalable-vector-struct.ll

```diff
@@ -1,23 +1,21 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
-; ifndef INTEL_SYCL_OPAQUEPOINTER_READY
```

Comment:
I thought we remove the code inside ifndef. Can you please clarify?

Thanks

Reply:
Yes, this was guarded because we were only testing typed pointer. This PR ported the testcase into opaque pointer, so we now can remove the guard to let it run.